about.css

```diff
@@ -0,0 +1,445 @@
+    height: 100%;
+    opacity: 0.2;
+    z-index: -1;
+    background-image: url('./img/ab-back\ -\ desk.png');
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-color: #fff;
+    padding-top: 50px;
+  }
+
+  .list-1 {
+    display: flex;
+    background-color: #000b;
+    justify-content: end;
+    color: white;
+    padding-right: 250px;
+    height: 25px;
+  }
+
+  ul {
+    list-style: none;
+  }
+
+  .list-1 ul {
+    display: flex;
+    align-items: center;
+    gap: 30px;
+  }
+
+  a {
+    text-decoration: none;
+  }
+
+  li a {
+    color: #fff;
+  }
+
+  .mobile-menu {
+    display: none;
+  }
+
+  .nav {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    gap: 40px;
+    background-color: #fff;
+    padding: 0 150px 0 150px;
+  }
+
+  .logo {
+    width: 70px;
+    height: 50px;
+  }
+
+  .nav ul {
+    display: flex;
+    align-items: center;
+    gap: 30px;
+  }
+
+  .list-h4 {
+    color: #ec5242;
+    border: 4px solid #ec5242;
+    padding: 5px;
+  }
+
+  .nav a:hover {
+    color: #ec5242;
+  }
+
+  .description {
+    display: flex;
+    flex-direction: column;
+    width: 800px;
+    margin: 100px 200px;
+    text-align: left;
+  }
+
+  .desc-header {
+    font-size: 25px;
+    color: #f00;
+  }
+
+  .description h1 {
+    font-size: 250%;
+    font-weight: 900;
+    color: #f00;
+    line-height: 55px;
+  }
+
+  .p-1 {
+    width: 500px;
+    background-color: #dddddded;
+    border: 3px solid white;
+    padding: 5px;
+    margin: 10px 5px;
+  }
+
+  .description .time {
+    margin-top: 20px;
+    font-weight: 900;
+  }
+
+  .venue {
+    font-weight: 600;
+    margin: 7px 0 200px;
+  }
+
+  .about-desc {
+    width: 100%;
+
+    /* height: 260px; */
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .about-desc p {
+    width: 760px;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 21px;
+  }
+
+  .block {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    border: 1px solid #ddd;
+    margin: 40px;
+  }
+
+  .past {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 40px;
+  }
+
+  .container {
+    display: flex;
+    align-items: center;
+    gap: 30px;
+  }
+
+  .img-container {
+    position: relative;
+    overflow: hidden;
+  }
+
+  .overlay {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 200px;
+    background-color: rgb(236, 82, 66, 0.5);
+    flex-direction: column;
+    justify-content: center;
+    color: #fff;
+    display: flex;
+    align-items: center;
+    gap: 10px;
+  }
+
+  .hide-desk {
+    display: none;
+  }
+
+  footer {
+    overflow: hidden;
+    width: 100%;
+    height: 245px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 30px;
+    padding: 0 100px;
+    margin-top: 70px;
+    background-color: #2e2e2e;
```

Comment:
- **[OPTIONAL]** I marked this requirement as OPTIONAL because it wasn't required but the last CR, but is a project requirement to follow the instruction from the original design. Please 🙏🏽 replace this color `#2e2e2e` for `#272A31`